cli/src/cordova.ts

```diff
@@ -291,9 +291,9 @@ export async function handleCordovaPluginsJS(
   config: Config,
   platform: string,
 ): Promise<void> {
-  if (!(await pathExists(await getWebDir(config, platform)))) {
-    await copyTask(config, platform);
```

Comment:
I believe we were running `copyTask` here simply so that the directory existed before we started writing the Cordova JS files, but I don't think we need to run a full copy, do we?

Reply:
It was sort of a fail safe in case the user cloned a repository and ran update without running copy first, not just to prevent the error.

But in this PR you have added the copyTask execution to both ios and android update commands if the folder doesn't exists, so, do we want that to happen?

Reply:
I think we do when running `update`. Eventually I'd like to see both `copy` and `update` deprecated and combined into `sync`. They're confusing and not very useful as separated commands IMO. Also, they do a bunch of stuff with Cordova that doesn't really fit into either.